lib/fathead/jmeter_docs/trigger_words.txt

```diff
@@ -0,0 +1,8 @@
+jmeter
+jmeter interface
+interface
```

Comment:
please remove the general terms, only triggers containing `jmeter` should be used